torch/csrc/jit/script/init.cpp

```diff
@@ -176,6 +176,10 @@ struct VISIBILITY_HIDDEN ConstantPythonTupleValue : public PythonValue {
     }
     return result;
   }
+
```

Comment:
This should be implemented as an override of `asValue` for ConstanPythonTuples, not as an additional method on the entire class hierarchy. This is what the `asValue` method is suppose to do. I don't think this will require any changes to `compiler.cpp` to accomplish.